modules/wallpapers/ReadMe.txt

```diff
@@ -28,4 +29,4 @@ To include the module in your build folder, run the configure script with the fl
 
 # For UNIX users
 
-    ./configure.sh -DENABLE_WALLPAPERS=1
\ No newline at end of file
+    ./configure.sh -DENABLE_WALLPAPERS=1
```

Comment:
What has been changed there?

Reply:
That line shouldn't be different, it might be a git bug.

Reply:
Ah, the original readme has no newline at the end, my editor appended it.